test/read-filestest.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const fs = require('fs');
+const expect = require('chai').expect;
+const myreadfile = require('../lib/read-files.js');
+
+let bacon = `${__dirname}/assets/bacon.txt`;
+let hipster = `${__dirname}/assets/hipster.txt`;
+let mobydick = `${__dirname}/assets/mobydick.txt`;
+
+
```

Comment:
Duncan helped Thomas out with how to properly test your function. I recommend viewing Thomas's PR and see his test module an replicate your test off that. 
[https://github.com/codefellows-seattle-javascript-401d14/lab-03-readfiles/pull/4/files](https://github.com/codefellows-seattle-javascript-401d14/lab-03-readfiles/pull/4/files)